pkgs/development/libraries/mesa/default.nix

```
@@ -227,6 +224,7 @@ in stdenv.mkDerivation {
   ] ++ lib.optionals needNativeCLC [
     (lib.mesonOption "intel-clc" "system")
```

```suggestion
    (lib.mesonOption "intel-clc" "system")
```

I believe intel-clc is no longer necessary, it's handled by precomp-compiler now. The two usage sites are [here][1] and [here][2]. I believe the plan is to remove it in the future. Same thing for `install-intel-clc` and `install-precomp-compiler`.

[1]: https://gitlab.freedesktop.org/mesa/mesa/-/blob/3a8abfa39b73d7fbb68492be864626fdff9c3ebc/meson.build#L323
[2]: https://gitlab.freedesktop.org/mesa/mesa/-/blob/3a8abfa39b73d7fbb68492be864626fdff9c3ebc/meson.build#L323

---

Fixed.